src/utils/getQuery/getQuery.ts

```diff
@@ -0,0 +1,16 @@
+import { SearchValues } from 'src/types';
+
+export const getAccommodationsListQuery = (queryParams: SearchValues): string => {
```

Comment:
let's rename it as `adaptParamsToURLEncoded`, we may need it on other API calls. Change file name accordingly

Reply:
Example code:
```
export const adaptParamsToURLEncoded = (params: Record<string, any>): URLSearchParams => {
  const URLSearchParamsInstance = new URLSearchParams();
  for (const key in params) {
    if (Object.prototype.hasOwnProperty.call(params, key)) {
      const value = params[key];
      if (value) {
        URLSearchParamsInstance.append(key, value);
      }
    }
  }
  return URLSearchParamsInstance;
};
```

And you can use it like this: 

```
 const response = await axiosInstance.post(
      SOME_ENDPOINT,
      adaptParamsToURLEncoded(params),
    );
```